spark/src/test/scala/org/apache/spark/sql/CometTestBase.scala

```diff
@@ -78,6 +78,7 @@ abstract class CometTestBase
     conf.set(CometConf.COMET_ENABLED.key, "true")
     conf.set(CometConf.COMET_EXEC_ENABLED.key, "true")
     conf.set(CometConf.COMET_EXEC_SHUFFLE_ENABLED.key, "true")
+    conf.set(CometConf.COMET_SHUFFLE_FALLBACK_TO_COLUMNAR.key, "true")
```

Comment:
Do we have a plan to enable this in Spark tests? I guess we need to fix some bugs first, but might be a good idea to file an issue ticket...

Reply:
The follow on issue for enabling it (actually removing the config) is https://github.com/apache/datafusion-comet/issues/1254

We need to fix the failures first: https://github.com/apache/datafusion-comet/issues/1256